### ISSUE TITLE
Dockerfile / Production setup configs

### ISSUE THREAD

**Issue by amiuhle** [amiuhle/kasisto#8]
* [ ] Create `Dockerfile` / `docker-compose.yml` to quickly start up development environment with `monerod` and `monero-wallet-rpc` running on testnet.
  This should somehow always be on `master` somehow, so there will need to be a way to download the latest successful build from https://build.getmonero.org/
* [ ] Derived from the above (or the other way round), a setup to run a permanent testnet node. Startup scripts for `monerod` and `monero-wallet-rpc` and cronjob for downloading and running latest build.
* [ ] Suggestions for a production setup, possibly different security levels:
  * [ ] Fool!
  * [ ] Default
  * [ ] Paranoid

**Comment by dternyak** [comment]
Is anyone working on this already? I may take a crack it at if not.

**Comment by amiuhle** [comment]
Not really. I've got a Dockerfile at https://github.com/amiuhle/monero-docker that will download the latest release which could be a starting point, but there are others out there could also be used.

What would be helpful most right now is to have a testnet `docker-compose.yml` in the project (`monerod`, `monero-wallet-rpc` with configurable wallet and password and `nginx` proxy for CORS headers). Testnet shouldn't be run from the latest stable release, however, but should follow the master branch. There is a buildbot, and builds end up here: https://build.getmonero.org/downloads/. There's no URL to get like the latest successful build for the master branch though. I brought that up on IRC, I'll check if there's any update on that.

I know the Dockerfiles could also compile from source, but having downloadable binaries as a starting point would mean we can reuse them for production setups on lower end hardware on which it would take a while to compile from source.

**Comment by amiuhle** [comment]
monero-project/monero/issues/1677

**Comment by malbit** [comment]
Hi. I would like to help you with that project and contribute in development. Im after such payment system like is in that repository.
